scriptlets/sru-helpers/fetch_manifest

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+#
+# Copyright 2024 Canonical Ltd.
+#
+# Fetch the manifest for a given machine from C3
+#
+# Usage:
+#     `fetch_manifest <CID> <MACHINE_SECURE_ID>`
+
+from argparse import ArgumentParser
+from io import StringIO
+import requests
+
+
+# This is the manifest that all machines will use if we cannot find
+# one specifically created for that machine
+fallback_manifest = """        [manifest]
+        com.canonical.certification::has_bt_smart = false
+        com.canonical.certification::has_camera = true
+        com.canonical.certification::has_card_reader = true
+        com.canonical.certification::has_ethernet_adapter = true
+        com.canonical.certification::has_thunderbolt = false
+        com.canonical.certification::has_thunderbolt3 = false
+        com.canonical.certification::has_touchscreen = false
+        com.canonical.certification::has_tpm2_chip = false
+        com.canonical.certification::has_usb_storage = true
+        com.canonical.certification::has_usb_type_c = false
+        com.canonical.certification::has_wlan_adapter = true
+"""
+
+
+def parse_json(manifest_json):
```

Comment:
I am pretty sure you can use `ConfigParser` for this sort of thing. https://docs.python.org/3/library/configparser.html

Reply:
Seems like `ConfigParser` is more powerful than I expected it to be. I have tried to incorporate all built-in methods in the script.